core/frontend/src/tile/map/MapTiledGraphicsProvider.ts

```diff
@@ -13,13 +13,13 @@ import { ViewState } from "../../ViewState";
 import { Viewport } from "../../Viewport";
 import { MapLayerImageryProvider, MapTileTreeReference, TiledGraphicsProvider, TileTreeReference } from "../internal";
 
-/** Position of a map-layer in the viewport
+/** Position of a map-layer in the display style's map (i.e. background/overlay map)
  * @beta */
 export interface MapLayerIndex {
-  // true if layer is overlay, otherwise layer is background.
+  /** true if map-layer is part of the display style's overlay map, otherwise map-layer is part of display style's background map */
   isOverlay: boolean;
 
-  // index of the map-layer
+  /** index of the map-layer in the display style's background/overlay map */
```

Comment:
Use documentation links please. I have an index - great. To what API(s) can I pass that index to obtain the thing identified by the index, or do things with the thing?

Reply:
the map-layer index is used in more than 10 methods of `DisplayStyleState`, so want all of them to be referenced?

Reply:
Probably not. The DisplayStyleState APIs aren't great...
Put yourself in the shoes of a reader trying to understand+use the API and document accordingly.
e.g., if I want to get the map layer at a particular index I call `DisplayStyleState.mapLayerAtIndex`, right?

Reply:
yes correct, I will at least reference this method.

Reply:
@pmconne how does it look now ?